loadable_apps/loadable_sample/Kconfig

```diff
@@ -77,6 +77,13 @@ config EXAMPLES_MICOM_TIMER_INTERVAL
 		Interval is based on microsecond.
 endif
 
+config WIFI_PF
```

Comment:
Why do you put this config in `Enable Test Scenarios` menu?
I think this config can be outside of test scenarios.